packages/palette_generator/lib/palette_generator.dart

```diff
@@ -37,6 +37,24 @@ class EncodedImage {
   final int height;
 }
 
+/// A class to hold the arguments for [PaletteGenerator.fromByteData] when
+/// calling from an isolate.
+class _FromByteDataArgs {
+  const _FromByteDataArgs(
+    this.encodedImage, {
+    this.region,
+    required this.maximumColorCount,
```

Comment:
We generally list required arguments first.

Reply:
Fixed in b2c6eb7d9.